src/OptiHPLCHandler/empower_api_core.py

```diff
@@ -10,6 +10,11 @@
 logger = logging.getLogger(__name__)
 
 
+class EmpowerResponse(NamedTuple):
+    result: Optional[dict]
```

Comment:
Is "result" the correct word? It is the relevant content of the response from the API. The "meat", so to speak, disregarding messages, HTTP codes etc.